ntbs-service/Pages/ServiceDirectory/Region.cshtml

```diff
@@ -0,0 +1,60 @@
+﻿@page "/ServiceDirectory/Region/{PhecCode}"
+@model ntbs_service.Pages.ServiceDirectory.RegionModel
+
+@{
+    Layout = "Shared/_Layout";
+}
+
+<nhs-width-container container-width="Standard">
+    <h2 class="nhsuk-heading-l">@Model.Phec?.Name</h2>
```

Comment:
Do we need a `?` here? surely this page only makes sense if backed by a `Phec` model?